onnx/defs/tensor/defs.cc

```diff
@@ -217,12 +217,13 @@ Takes a  parameter `axes` with a list of axes to squeeze.
             "Constrain input and output types to float tensors.");
 
 OPERATOR_SCHEMA(Pad)
+    .SinceVersion(2)
     .NumInputs(1)
     .NumOutputs(1)
-    .Attr("paddings",
+    .Attr("pads",
```

Comment:
Backend test also needs to be updated: https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/pad.py#L22

Reply:
Done.

Reply:
After updating the test cases python/numpy code, we need to run `backend-test-tools generate-data` to regenerate the test data. I have just done it for this PR.

Reply:
Thanks @bddppq  do we have that written somewhere?

Reply:
Yep I added this doc recently: https://github.com/onnx/onnx/blob/master/docs/ONNX%20Backend%20Test.md